scripts/migrations/migrate_isoflop_checkpoints.sh

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env bash
+# Migrate legacy ISOFlop checkpoints into the static isoflop/ output tree.
```

Comment:
What's the thinking behind checking in this script OOC @Helw150?  Should it ever be needed again?

Reply:
It shouldn't be needed again, no. This is mostly muscle memory from old employment norms (DB migration scripts always got code reviewed and checked in so they could be rolled back).

Not sure what we want out norm to be!

Reply:
It shouldn't be needed again, no. This is mostly muscle memory from old employment norms (DB migration scripts always got code reviewed and checked in so they could be rolled back).

Not sure what we want out norm to be!